pram

```diff
@@ -21,6 +21,8 @@ print_help() {
 	echo "Options:"
 	echo "  --am-options OPTIONS"
 	echo "                   Pass additional OPTIONS to git am"
+	echo "  --api            Use the GitHub REST API and authenticate with"
```

Comment:
Ah, that's what you meant. I guess it's fine, though I'd still enable it by default when token is available.

Reply:
Changed it to be default on if token present with the ability to disable with --no-api.